package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "hackton-backend",
+  "version": "1.0.0",
+  "description": "This is a Lambda EU3 Labs project\r # hackton-server\r A comprehensive portal for hackathons where students can submit their work and judges can evaluate in one place",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node index.js",
+    "server": "nodemon index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/LABS-EU3/hackton-backend.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/LABS-EU3/hackton-backend/issues"
+  },
+  "homepage": "https://github.com/LABS-EU3/hackton-backend#readme",
+  "dependencies": {
+    "cors": "^2.8.5",
```

Comment:
Missing `pg` package

Reply:
It should be part of the initial setup IMO

Reply:
I'll add it then